pylintrc_tweaks

```diff
@@ -0,0 +1,5 @@
+[MASTER]
+load-plugins = edx_lint.pylint
+
+[MESSAGES CONTROL]
+DISABLE+= ,literal-used-as-attribute
```

Comment:
I am not too familiar with this code base. The calls where this error occurred seem legitimate, so I opted not to change them. 